deployments/warp_routes/KLC/logo.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="504" height="505" xml:space="preserve" version="1.1" viewBox="0 0 504 505">
```

Comment:
same comment as before, it's just a png with svg tags

Reply:
applied your suggested change